BUILD.bazel

```diff
@@ -9,7 +9,6 @@ filegroup(
     srcs = [
         ".bazelrc",
         ".gitignore",
-        ".user.bazelrc",
```

Comment:
This is being included because of the gazelle `filegroup` extension that we are using.

So you should be adding

```
# gazelle:exclude .user.bazelrc
```

to the top of this file for it to properly get excluded.

Thanks,